metaworld/envs/mujoco/sawyer_xyz/sawyer_shelf_place.py

```diff
@@ -103,6 +105,7 @@ def reset_model(self):
         self._set_obj_xyz(self.obj_init_pos)
         self.maxPlacingDist = np.linalg.norm(np.array([self.obj_init_pos[0], self.obj_init_pos[1], self.heightTarget]) - np.array(self._state_goal)) + self.heightTarget
         self.target_reward = 1000*self.maxPlacingDist + 1000*2
+        self.num_resets += 1
```

Comment:
What is this for?

Reply:
I found this in the other environments and it wasn't present in shelf so I just added it. I shelf had some things leftover from the observation space fix and new API that hadn't been updated so I just tried to update it as best I could